staging/src/k8s.io/kubectl/pkg/cmd/label/label_test.go

```diff
@@ -696,18 +699,20 @@ pod/foo not labeled
 			if tc.dryRun != "" {
 				cmd.Flags().Set("dry-run", tc.dryRun)
 			}
-			options := NewLabelOptions(iostreams)
+
+			flags := NewLabelFlags(tf, iostreams)
+			options, err := flags.ToOptions(cmd, tc.args)
 			if tc.overwrite {
 				options.overwrite = true
 			}
-			if err := options.Complete(tf, cmd, tc.args); err != nil {
+			if err != nil {
```

Comment:
if options is nil, above assignment will panick `options.ovewrite = true`. Maybe we can change the order and first check `err != nil` before assignment?

Reply:
Well noticed. Thanks!